synthetics/test-data.json

```diff
@@ -95,14 +95,10 @@
           "server-one",
           "server-two"
         ],
-        "targetType": "dummy-tt",
-        "targetValue": "dummy-tv",
-        "useLocalIp": false,
```

Comment:
I'd prefer you not to modify lines other than those with targetType and targetValue.
This file is based on real payloads returned by the server, so modifying them would make it less realistic. I have no strong opinion about that, tough.

Reply:
After `make test` this fille is changed automatically in many places. I should ignore those changes and commit its state before i run `make test`?

Reply:
Yes

Reply:
Addressed with latest commit